java/src/jmri/script/ScriptEngineSelector.java

```diff
@@ -114,6 +114,15 @@ public boolean isJython() {
                     || "jython".equalsIgnoreCase(_languageName);
         }
 
+        /**
+         * Is this engine for JavaScript/ECMAscript?
+         * @return true if JavaScript/ECMAscript, false otherwise
+         */
+        public boolean isJavaScript() {
```

Comment:
No need to change. But what's the purpose of this method? It doesn't seem to be used anywhere.